src/Libraries/RevitServices/Elements/ModelUpdater.cs

```diff
@@ -29,36 +29,50 @@ public class RevitServicesUpdater
 
         private readonly HashSet<IUpdater> registeredUpdaters;
 
+        [Obsolete("Use ElementsUpdated event")]
```

Comment:
so to clarify, these events are still raised so old clients can still use them, but they are no longer subscribed to anywhere in in this codebase?